mozilla-release/browser/app/nsBrowserApp.cpp

```diff
@@ -84,9 +84,9 @@ static void Output(const char *fmt, ... )
     decltype(MessageBoxW)* messageBoxW =
       (decltype(MessageBoxW)*) GetProcAddress(user32, "MessageBoxW");
     if (messageBoxW) {
-      messageBoxW(nullptr, wide_msg, L"Firefox", MB_OK
-                                               | MB_ICONERROR
-                                               | MB_SETFOREGROUND);
+      // TODO: Use `sAppData->name`
+      messageBoxW(nullptr, wide_msg, L"CLIQZ",
+                  MB_OK | MB_ICONERROR | MB_SETFOREGROUND);
```

Comment:
This one isn't actually required by DB-312, but I decided to make a preventive fix.